src/libs/SearchUtils.ts

```diff
@@ -254,8 +258,8 @@ function getSortedTransactionData(data: TransactionListItemType[], sortBy?: Sear
     }
 
     return data.sort((a, b) => {
-        const aValue = a[sortingProperty];
-        const bValue = b[sortingProperty];
+        const aValue = getNestedValue(a, sortingProperty);
+        const bValue = getNestedValue(b, sortingProperty);
```

Comment:
```suggestion
        const aValue = sortingProperty === 'comment' ? a.comment.comment : a[sortingProperty];
        const bValue = sortingProperty === 'comment' ? b.comment.comment : b[sortingProperty];
```

Your solution is generic but it's causing some ts errors. Do you think we can go with a simple `sortingProperty` check? This will also avoid calling `reduce` in a loop

Reply:
I think that's fine for now and would avoid the reduce loop. We can revisit a more generic solution if we end up with more nested properties